lib/cryptoexchange/helpers/token_helper.rb

```diff
@@ -0,0 +1,22 @@
+module TokenHelper
+  class << self
+    API_URL = 'https://api.ethplorer.io/getTokenInfo'
+    def get_symbol(token)
+      endpoint = "#{API_URL}/#{token}?apiKey=buoc5056EEZGf105"
```

Comment:
@Swingcloud Not too good having an API key dropped in here, we should make it configurable from the user end. Consider using a config file